src/main/java/com/epam/reportportal/cucumber/AbstractReporter.java

```diff
@@ -259,6 +274,7 @@ protected void beforeScenario(Scenario scenario, String outlineIteration) {
 		scenarioContext.setId(startScenario(featureContext.getId(), rq));
 		scenarioContext.setLine(scenario.getLine());
 		scenarioContext.setFeatureUri(uri);
+		descriptionsMap.put(scenarioContext.getId().blockingGet(), ofNullable(rq.getDescription()).orElse(StringUtils.EMPTY));
```

Comment:
I don not allow to use `blockingGet` here, this will break our asynchronous processing, please rewrite.

Reply:
The issue was fixed in commit b4c87fd1b8efa718a2e6076370b40a3b1e3274c8
Thank you !